nucypher/blockchain/eth/sol/source/zeppelin/utils/Address.sol

```diff
@@ -0,0 +1,26 @@
+pragma solidity ^0.5.3;
+
+/**
+ * Utility library of inline functions on addresses
+ */
+library Address {
+    /**
+     * Returns whether the target address is a contract
+     * @dev This function will return false if invoked during the constructor of a contract,
+     * as the code is not actually created until after the constructor finishes.
+     * @param account address of the account to check
+     * @return whether the target address is a contract
+     */
+    function isContract(address account) internal view returns (bool) {
+        uint256 size;
+        // XXX Currently there is no better way to check if there is a contract in an address
+        // than to check the size of the code at that address.
+        // See https://ethereum.stackexchange.com/a/14016/36603
+        // for more details about how this works.
+        // TODO Check this again before the Serenity release, because all addresses will be
```

Comment:
This is really interesting and unfortunate. I guess we'll be waiting until 2021?

Reply:
I think so, I tried not to use this method, so mostly to check for contract - used call some method.